Verify the support of the led trigger input source and complain in a helpful
way if modules are missing during bootup.

Berichtet via openwrt-devel@lists.openwrt.org am 29.12.2014
--- a/openwrt/package/base-files/files/etc/init.d/led
+++ b/openwrt/package/base-files/files/etc/init.d/led
@@ -40,7 +40,11 @@ load_led() {
 			[ $default -eq 1 ] ||
 				echo 0 >/sys/class/leds/${sysfs}/brightness
 		}
-		echo $trigger > /sys/class/leds/${sysfs}/trigger
+		# this may fail if the module is not loaded
+		if ! echo $trigger > /sys/class/leds/${sysfs}/trigger 2>/dev/null; then
+			echo >&2 "Skipping trigger '$trigger' for led '$name' due to missing kernel module"
+			return 0
+		fi
 		case "$trigger" in
 		"netdev")
 			[ -n "$dev" ] && {
